lib/msf/core/web_services/module_search.rb

```diff
@@ -0,0 +1,72 @@
+module Msf::WebServices::ModuleSearch
+
+  def search_modules(opts)
+    raise ::ArgumentError, "At least one search parameter must be provided." if opts.dup.except!(:fields).empty?
+    search_results = []
+    metadata = Msf::Modules::Metadata::Cache.instance.get_metadata
+    params = parse_params(opts)
+    metadata.each { |module_metadata|
```

Comment:
Recommend using the following format for multi-line blocks. [Examples](https://github.com/rubocop-hq/ruby-style-guide#single-line-blocks)

```
metadata.each do |module_metadata
...
end
```

Reply:
👍